Базовая стилизация всех страниц

Callipsoul opened the pull request:
---
:mortar_board: [Базовая стилизация всех страниц](https://up.htmlacademy.ru/htmlcss-individual/1/user/2498751/tasks/6)

Callipsoul committed:
Базовая стилизация всех страниц

ustimenko-andrew commented on styles/styles.css:
```diff
@@ -169,7 +226,7 @@ body {
 }
 
-.button-blue {
+.button-big-blue {
```
Старайся писать абстрактные имена, то есть не blue, а допустим primary или что-то, что не связано напрямую с отображением

ustimenko-andrew commented on styles/styles.css:
```diff
@@ -180,6 +237,16 @@ body {
+  display: flex;
+}
+
+.footer-social-list {
```
Попробуй выделить общие классы ( для кнопок, списков ), это поможет тебе в будущем их переспользовать в разных элементах. 
Допустим .footer-social-list  может легко стать list-reset, где ты с 99% вероятностью уберешь margin, padding, list-style. Подобное будет и со всеми кнопками.

ustimenko-andrew commented on styles/styles.css:
```diff
@@ -199,3 +266,150 @@ body {
+}
+
+.button {
+  text-align: center;
```
Это значение по умолчанию у кнопок, перепроверь классы на лишние стили, значения по умолчанию, для чистоты кода, не стоит дублировать

Callipsoul committed:
Правки по базовой стилизации

Добавила переменные для шрифтов, общие классы для кнопок и списков.

Callipsoul committed:
Правки по базовой стилизации 2

Убрала класс button у ссылок.
Убрала font-style везде, тк он не меняется.
Создала общий класс для всех ссылок и прописала там text-decoration.
Сделала общий класс для advantages-item с одним цветом фона.
field=>email-field.
Поправила названия переменных для шрифтов.
